presto-main/src/main/java/com/facebook/presto/metadata/FunctionRegistry.java

```diff
@@ -609,7 +610,8 @@ public FunctionRegistry(TypeManager typeManager, BlockEncodingSerde blockEncodin
                 .aggregate(BuildSetDigestAggregation.class)
                 .scalars(SetDigestFunctions.class)
                 .scalars(SetDigestOperators.class)
-                .scalars(WilsonInterval.class);
+                .scalars(WilsonInterval.class)
+                .scalars(DataSizeFunctions.class);
```

Comment:
Let's put this above (around line 500), with most of the other scalars. (this whole list needs reorganization)